USER_MANUAL.md

```diff
@@ -3042,6 +3042,84 @@ Using `rbreak <regex>` you can set breakpoints on multiple functions.
 -   `(gdb) break definition.kore:break -> No source file named definition.kore.`
 send `-ccopt -g` to kompile in order to generate debug info symbols.
 
+Profiling your K semantics
+--------------------------
+
+The first thing to be aware of is in order to get meaningful data,
+you need to build the semantics and all of its dependencies with
+optimizations enabled but _without the frame pointer elimination
+optimization_. For example, for EVM, this means rebuilding GMP, MPFR,
+JEMalloc, Crypto++, SECP256K1, etc with the following `exports`. 
+```shell
+  export CFLAGS="-g -O2 -fno-omit-frame-pointer" ;
+  export CXXFLAGS="-DNDEBUG -O2 -fno-omit-frame-pointer"
+```
+
+You can skip this step, but if you do, any samples within these
+libraries will not have correct stack trace information, which means
```

Comment:
Look at the rest of this flie, and format this the same way as the rest.

A space before the backticks codeblock, a space after it, no spaces before each line in the code block, etc...

You should _always_ inspect the formatting of th erest of the file or project or whatever you're contributing to, it's up to you, the PR author, to make sure this is correct. Burden should not be shifted onto the reviewers.